### PR TITLE
Feat/close button on tiles

### PR DESCRIPTION
**Pull request by mpisacic**

### CODE REVIEW

File: examples/src/CompactServer.tsx [hunk 1]
```diff
@@ -13,8 +13,8 @@ export class Index extends React.Component<any, any> {
...
-                <CompactServer id={'My very very long name of a server I am using I know its very long.domain.com' } onClose={this._onServerCloseCompactServer} onRoleEdit={this._onClickCompactServer} name={'My very very long name of a server I am using I know its very long'} roles={[]} status={2} />
-                <CompactServer id={'BANANA-PC.banana.com' } onClose={this._onServerCloseCompactServer} onRoleEdit={this._onClickCompactServer} name={'BANANA-PC'} roles={[{ display: 'WPF', iconName: 'icon-add' }, { display: 'Search', iconName: 'icon-alert' }]} status={0} />
+                <CompactServer id={'My very very long name of a server I am using I know its very long.domain.com' } onRoleEdit={this._onClickCompactServer} name={'My very very long name of a server I am using I know its very long'} roles={[]} status={2} />
+                <CompactServer id={'BANANA-PC.banana.com' } onRoleEdit={this._onClickCompactServer} onClose={this._onServerCloseCompactServer} name={'BANANA-PC'} roles={[{ display: 'WPF', iconName: 'icon-add' }, { display: 'Search', iconName: 'icon-alert' }]} status={0} />
```
Comment (vinkobedek) [inline examples/src/CompactServer.tsx]: reading this  *onRoleEdit={this._onClickCompactServer}* confuses me a bit. Are we editing roles, or are we clicking the server?

File: examples/src/Dashboard.tsx [hunk 2]
```diff
@@ -66,7 +66,7 @@ export class Index extends React.Component<any, any> {
                     groupAddFunc={dummyDashboard.groupAddFunc}
                     groupDeleteFunc={dummyDashboard.groupDeleteFunc}
                     groupEditFunc={dummyDashboard.groupEditFunc}
-                    serverClose={dummyDashboard.serverClose}
+                    serverClose={(serverId, groupId) => console.log('removing server : ', serverId, groupId)}
```
Comment (vinkobedek) [inline examples/src/Dashboard.tsx]: what happened here? was it meant to be temporary?

Comment (mpisacic) [inline examples/src/Dashboard.tsx]: it's a typo, forgot to bind to prop will do it now

File: examples/src/CompactServer.tsx (hunk 1 above)
Comment (mpisacic) [inline examples/src/CompactServer.tsx]: changed function name to _onRoleEit

File: src/components/CompactFarm/CompactFarm.tsx [hunk 3]
```diff
@@ -122,7 +122,7 @@ export class CompactFarm extends CommonComponent<ICompactFarmProps, any> {
                                 status={server.status}
                                 roleEdit={this._onRoleChange}
                                 showEditRoles={this.props.showEditRoles}
-                                onClose={server.onClose}
+                                onClose={this.props.serverClose !== undefined ? this._onServerClose : undefined}
```
Comment (vinkobedek) [inline src/components/CompactFarm/CompactFarm.tsx]: personally i like the **on** prefix in the prop names when dealing with events. I can tell immediately that it is some event handler

**Pull request merged**